ticket_sales/templates/index.html

```diff
@@ -72,13 +73,13 @@ <h3 style="font-family: b yekan;">آخرین <span>بازی</span></h3>
                                 <div class="uk-grid uk-flex uk-flex-middle">
                                     <div class="uk-width-2-12 center">
                                         <a href="#">
-                                            <img src="{% static 'images/club-logo.png' %}" class="img-polaroid" alt=""
+                                            <img src="{{ event.home_team.logo.url }}" class="img-polaroid" alt=""
                                                  title="">
                                         </a>
                                     </div>
                                     <div class="uk-width-3-12 name uk-vertical-align">
                                         <div class="wrap uk-vertical-align-middle">
-                                            مس کرمان
+                                            {{ event.home_team }}
```

Comment:
You should show the Team's name, not the Team str method.
```suggestion
                                            {{ event.home_team.name }}
```